ci/release/update-version.sh

```diff
@@ -21,22 +21,18 @@ NEXT_FULL_TAG=$1
 
 # Get current version
 CURRENT_TAG=$(git tag --merged HEAD | grep -xE '^v.*' | sort --version-sort | tail -n 1 | tr -d 'v')
-CURRENT_MAJOR=$(echo $CURRENT_TAG | awk '{split($0, a, "."); print a[1]}')
```

Comment:
Why are the current variables not needed anymore? Just wondering.

Reply:
Probably not used anymore.  One of the advantages of a tool like this is it can identify when we've missed cleaning up things when the scripts change.

Reply:
Yeah @ChuckHastings has it right, it just isn't used anywhere.